### PR TITLE
Replace sum by np.sum in NB to fix minimizing error

### PR DESCRIPTION
**Pull request by combet**
This addresses #412. There already was an approved PR for this but from a branch created from an erroneous `master` branch (that is now fixed). So repeating this but from a new branch, created from the fixed `master` to avoid any problems.

### CODE REVIEW

**Comment by combet** [comment]
@m-aguena - I've created a new branch for this small fix, from the `master` branch you fixed yesterday as the originally approved PR was from a branch created from the erroneous `master` (it reported 15 files changed while this was just a one line fix). Might have been another way to do it, but thought it was cleaner to start again. I've closed the previous (unmerged) PR and will merge this one, once you approve it again :)

**Review by m-aguena — APPROVED** [review]

**Pull request merged**
